bowling-ball/Frames.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingBall
+{
+    class Frames : IFrames
+    {
+        int[] EachFrame = new int[2];
+        const int LowestPossibleValue = 0;
+        const int HighestPossibleValue = 10;
+        const int EmptyFrame = -1;
+
+        public Frames()
+        {
+            this.EachFrame[0] = -1;
+            this.EachFrame[1] = -1;
+        }
+
+        public void SetFrame(int pin, int indexStatus) => this.EachFrame[indexStatus] = pin;

```

Comment:
`indexStatus`? is it a status or just an index? avoid using confusing names
also, index should appear as first param.